app/models/book.rb

```diff
@@ -1,3 +1,3 @@
-class Book < ActiveRecord::Base
-    serialize :params
+class Book < ApplicationRecord
+  serialize :params
```

Comment:
The card tells you to use 
https://github.com/rails-api/active_model_serializers